### ISSUE TITLE
Selvidentifiserte brukere

### ISSUE THREAD

**Issue by ekorra** [Altinn/altinn-authentication#218]
### Overordnet beskrivelse av behov

Det er behov for en bedre løsning for brukere uten Dnr/Fnr. Ønsker ikke at dette skal være en Altinn intern bruker, men en bruker som kan brukes på tvers av offentlig sektor.
Det må være mulig å delegere til og fra en selvidentifisert bruker

Løsningen må sikre at dagens SI-brukere kan beholde tilgangen til de meldinger osv. de har i Altinn fra før
 
 
### Spørsmål
Hvordan kobler vi dagnes SI brukere til ny nasjonal løsning (ID-porten utland el)

**Comment by annerisbakk** [comment]
ID-porten skal lage en løsning som overtar selvidentifiserte brukere. Gjenstår løsning for å koble eksisterende selvid.brukere til ID-porten

**Comment by ekorra** [comment]
Ja. Selvregistrert bruker (SI-bruikker) vil erstattes av e-post bruker fra ID-porten. 
Rune legger til rette for å kunne logge på med denne brukeren i forbindelse med OIDC arbeidet han gjør nå. Etter den dele er ferdig må vi utvikle noe slik at man kan koble historikken man har med en SI-bruker i Altinn2 med epost-brukeren.
